### PR TITLE
Fixed "Show in folder" for paths with comma

### PR DESCRIPTION
**Pull request by m417z**
A simple fix. I didn't test the code, but I tested the command line and it works.

### CODE REVIEW

File: GitUI/OsShellUtil.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ public static void OpenAs(string file)
 
         public static void SelectPathInFileExplorer(string filePath)
         {
-            Process.Start("explorer.exe", "/select, " + filePath);
+            Process.Start("explorer.exe", "/select, \"" + filePath + "\"");
```
Comment (RussKie) [inline GitUI/OsShellUtil.cs:21]: We have an extension method
```suggestion
            Process.Start("explorer.exe", $"/select, {filePath.Quote()}");
```

**Comment by RussKie** [comment]
Please provide your sign off in the contributors.txt

File: contributors.txt [hunk 2]
```diff
@@ -139,3 +139,4 @@ YYYY/MM/DD, github id, Full name, email
 2020/09/09, adamshakhabov, Adam Shakhabov, theadamo86(at)gmail.com
 2020/09/13, hoborm, Mihaly Hobor, mail(at)hobor.xyz
 2020/07/11, blitzmann, Ryan Holmes, holmes.ryan.90(at)gmail.com
+2020/10/03, m417z, ,
```
Comment (gerhardol) [inline contributors.txt:142]: Name and email is required (nor use special or ask someone else to use the patch)

**Review by gerhardol — CHANGES_REQUESTED** [review]

**Comment by RussKie** [comment]
I've tested the current release, and I don't believe there is a problem selecting a file:

image
image

**Comment by m417z** [comment]
@RussKie you're right, the bug doesn't happen with a space. It does happen when a path contains a comma, though.
For example, this doesn't work: `explorer.exe /select, C:\test,test`
This works (assuming the folder exists): `explorer.exe /select, "C:\test,test"`

**Pull request closed without merging**